### PR TITLE
feat: Add login option in the overflow menu

### LINKED ISSUE #1703: Add signin / signup option in the menu
**Actual Behaviour**

User is not able to sign in / sign up later if he skips the sign in /sign up process when the app starts for the first time

**Expected Behaviour**

User must be able to sign in / sign up anytime he wants

**Steps to reproduce it**

Option for sign in / sign up can be added to  the menu  which will take the user to sign in page

**Would you like to work on the issue?**

Yes , I would like to work on this issue

### PR DESCRIPTION
**Pull request by Princejhabak**
Fixes #1703

Changes: Login option added to the overflow menu which will take the user to the login screen

Screenshots for the change:
46090175-c7d6ce00-c1cd-11e8-9aac-11fdfb8fcb0c

### CODE REVIEW

**Comment by Princejhabak** [comment]
@arundhati24 @iamareebjamal Please review my PR again.

File: app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt [hunk 1]
```diff
@@ -60,6 +64,12 @@ class SkillsActivity : AppCompatActivity(), SkillFragmentCallback {
     override fun onCreateOptionsMenu(menu: Menu?): Boolean {
         val menuInflater = menuInflater
         menuInflater.inflate(R.menu.skills_activity_menu, menu)
+
+        if (PrefManager.getBoolean("logged_in", false)){
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt]: Space after `)`

**Comment by Princejhabak** [comment]
@iamareebjamal  I've added space after ) please review my PR again.

**Comment by Princejhabak** [comment]
@arundhati24 The overflow menu will not show login option if the user is already logged in.

**Comment by Princejhabak** [comment]
@arundhati24 @batbrain7, I tried to work on log out option but I could make it work. I couldn't figure out how the logout process works. I can intent the user to the log in screen if he clicks on log out option but it will not actually log out the user. So what should I do now ??

**Comment by Nimagna123** [comment]
Is this issue closed or should i start working for adding logout option and login option ?

**Comment by NileshPranami** [comment]
I am about to resolve this issue

**Comment by immadisairaj** [comment]
@iamareebjamal @batbrain7 
I think we can close this PR as this is already fixed in the PR #2046

**Pull request closed without merging**